cli/cmd/upgrade/upgrade.go

```diff
@@ -170,11 +168,18 @@ func (c *Command) Run(args []string) int {
 
 	defer common.CloseWithError(c.BaseCommand)
 
-	if err := c.validateFlags(args); err != nil {
+	err := c.validateFlags(args)
+	if err != nil {
```

Comment:
This is kind of a knock-on change from the change below at line 177. If I use `:=` on line 177 the compiler gets mad because `c.timeoutDuration` is already instantiated (even though err isn't). So I needed to instantiate `err` in the function scope to fix that error. I could do that either with the `var` keyword or just by moving this check. Both added a new line and I felt that between the two, this was the cleaner option. Otherwise there would be an `err` var shadowing an `err` in the enclosing scope that wouldn't be instantiated until later.